ui/pages/Project/constants.js

```diff
@@ -195,26 +195,33 @@ export const CATEGORY_FAMILY_FILTERS = {
     {
       value: SHOW_ANALYSED_BY_ME,
       name: 'Analysed By Me',
-      createFilter: (individualsByGuid, user) => family => family.analysedBy.some(
-        analysedBy => analysedBy.createdBy === (user.displayName || user.email),
-      ),
-    },
-    {
-      value: SHOW_NOT_ANALYSED_BY_ME,
-      name: 'Not Analysed By Me',
-      createFilter: (individualsByGuid, user) => family => family.analysedBy.every(
-        analysedBy => analysedBy.createdBy !== (user.displayName || user.email),
-      ),
+      analysedByFilter: (individualsByGuid, user) => ({ createdBy }) => createdBy === (user.displayName || user.email),
     },
     {
       value: SHOW_ANALYSED,
       name: 'Analysed',
-      createFilter: () => family => family.analysedBy.length > 0,
+      analysedByFilter: () => () => true,
     },
     {
       value: SHOW_NOT_ANALYSED,
       name: 'Not Analysed',
-      createFilter: () => family => family.analysedBy.length < 1,
+      requireNoAnalysedBy: true,
+      analysedByFilter: () => () => true,
+    },
+    ...FAMILY_ANALYSED_BY_DATA_TYPES.map(([type, typeDisplay]) => ({
+      value: type,
+      name: typeDisplay,
+      category: 'Data Type',
+      analysedByFilter: () => ({ dataType }) => dataType === type,
+    })),
+    {
+      value: 'yearSinceAnalysed',
+      name: '>1 Year',
+      category: 'Analysis Date',
+      requireNoAnalysedBy: true,
+      analysedByFilter: () => ({ lastModifiedDate }) => (
+        (new Date()).setFullYear(new Date().getFullYear() - 1) < new Date(lastModifiedDate)
+      ),
```

Comment:
Do we need to add a space or a separator between categories (e.g. `Data Type`, `Analysis Date`, and `Analysed By`)?

Reply:
the way the dropdown shows the different categories automatically adds a category header and divider between the sections